index.html

```diff
@@ -158,8 +158,15 @@
            observer.disconnect();
       }
     });
-    // subscribe to Resource-Timing and User-Timing events
-    observer.observe({entryTypes: ['resource', 'mark', 'measure']});
+    observer.observe({
+      // subscribe to Resource-Timing, User-Timing, and Server-Timing events
+      entryTypes:
+        ['resource', 'mark', 'measure', 'server'],
+
+      // retrieve the buffered Resource-Timing, User-Timing, and Server-Timing events
+      bufferedTypes:
+        ['resource', 'mark', 'measure', 'server']
```

Comment:
Ah, ok, I think I see the confusion. I'll try to unpack..

1) We _should not_ affect RT/UT processing in any way: all existing entries should be added to the timeline as usual, with no new restrictions on before/after onload.
2) I'm proposing that we buffer **all** entry types up until the end of onload event.
   - This doesn't affect current entries like RT/UT, as that's already the case.
   - This does mean that any+all PO-only events are buffered until onload: ST, Long Tasks, and so on.

As a result, entries that have their own buffers are limited by capacity of those buffers, and entries that are queue via PerfObserver are only buffered until end of onload. 

### Retrieving entries
- Entries that are queued into global performance timeline can be retrieved via `performance.getEntriesBy{Name/Type}`. This falls out naturally and works for all types, with only difference that PO-queued entries are only buffered up until end of onload.
- When a new PO is registered and provides `buffer:true`, under the hood we pull out all entries of the specified types from the global perf timeline and queue them into the PO observer. In that sense, it's roughly the same as "register and then immediately call getEntries" but avoids race conditions and provides better developer ergonomics.

---

Is that coherent?